### DISCUSSION TITLE
Welcome to Mastodon.swift Discussions!

### DISCUSSION THREAD

**Discussion by DK1DA** [Swiftodon/Mastodon.swift#25] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by michaelrowe01** [comment]
Hi I am Michael, I’ve been playing with Swift for a long time and am always interested in learning more.  I have an app on the AppStore that I would like to add in support for Mastodon, and glad to have found this project.  Hope I can help in some way.
